doc/RELEASE_NOTES.md

```diff
@@ -203,16 +197,16 @@ As such, the minimum supported version of RHEL and CentOS is 7.9.
 
 ## Support for `pbench-trafficgen` dropped entirely
 
-With the release of v0.71 support for `pbench-trafficgen` has been removed in its entirety. The ability to support the behavior of that benchmark is too difficult given the implementation of `pbench-trafficgen` and `pbench-run-benchmark`'s trafficgen support.
+With the release of v0.71, support for `pbench-trafficgen` has been removed in its entirety. It is too difficult to support the behavior of that benchmark workload given the implementation of `pbench-trafficgen` and `pbench-run-benchmark`'s trafficgen support.
 
-Future work on supporting benchmarks will be approached by working to have the Pbench Agent integrate with separate software packages that are dedicated to running benchmarks (unlike the Pbench Agent which only provides convenience interfaces).
+Future work on supporting benchmark workloads will be approached by working to have the Pbench Agent integrate with separate software packages that are dedicated to running benchmarks (unlike the Pbench Agent which only provides convenience interfaces).
 
 
 ## Deprecation Notices and Removals
 
 ### Deprecated Bench Scripts
 
-The following `bench-scripts` have been deprecated with this release, and will be removed entirely in the next release:
+The following `bench-scripts` have been deprecated with this release and will be removed entirely in the next release:
```

Comment:
I'm not sure "bench-scripts" should be in code format, unless you want to be more specific, e.g., "scripts in the `/opt/pbench-agent/bench-scripts` directory"?